job-app/src/App.js

```diff
@@ -117,7 +134,8 @@ class App extends React.Component {
                             </div>
                         </Container>
                         <div className="options">
-                            <Button variant="primary" onClick={() => { this.setState({popupOpenAdd: true}); }}>+ New Application</Button> {' '}
+                            <Button variant="primary" className="btn-primary" onClick={() => { this.setState({popupOpen: true}); }}>+ New Application</Button> {' '}
```

Comment:
I don't think you need to do anything, but just wanted to note that this should include "popupOpenAdd" in master

Reply:
@randallkentbrown